src/main/java/com/sap/sgs/phosphor/fosstars/data/github/NvdEntryMatcher.java

```diff
@@ -87,30 +106,14 @@ private boolean match(Configurations configurations) {
       return false;
     }
 
-    for (Node node : configurations.getNodes()) {
-      if (node.getCpeMatches() == null) {
-        continue;
-      }
-
-      for (CpeMatch cpeMatch : node.getCpeMatches()) {
-        if (cpeMatch == null) {
-          continue;
-        }
-
-        if (match(cpeMatch.getCpe22Uri()) || match(cpeMatch.getCpe23Uri())) {
-          return true;
-        }
-      }
-    }
-
-    return false;
+    return parseNodes(configurations.getNodes());
   }
 
   /**
    * Returns true if a string matches the project, false otherwise.
    */
   private boolean match(String string) {
-    return string != null && string.toLowerCase().contains(project.name());
+    return match(project.name(), string.toLowerCase().trim());
```

Comment:
Here we are anyway doing equalsIgnoreCase https://github.com/SAP/fosstars-rating-core/pull/180/files#diff-399975d627508494b618551e2122f721R180 then why do we need string.toLowerCase() again?